src/main/scala/wordbots/Lexicon.scala

```diff
@@ -68,9 +69,11 @@ object Lexicon {
       (S\NP, λ {t: TargetObject => AfterAttack(t, AllObjects)}),
       ((S\NP)/N, λ {o: ObjectType => λ {t: TargetObject => AfterAttack(t, o)}})
     )) +
+  //("attack" -> see "power")+
```

Comment:
Honestly, I never really use the word "power", so maybe the `Seq(power, attack)` entry should be flipped to `Seq(attack, power)` and moved up here.